cron/daily/make-private-updates-public.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+import models, { Op } from '../../server/models';
+import logger from '../../server/lib/logger';
+
+// get the ISOtime for the day @ midnight to compare
+// against the update.makePublic on time field
+
+const today = new Date(new Date().setUTCHours(0, 0, 0, 0)).toISOString();
+logger.info(`Updates for ${today}`);
+
+// get all the private updates with the makePublicOn <= today
+// and make them public
+
+let updates = 0;
+
+models.Update.findAll({
```

Comment:
Updating row per row makes the code pretty verbose and it's not performant: if we have 100 updates to do, we'll trigger 100 SQL queries.

A more concise way to write this would be to do something like:

```es6
models.Update.update(
  {
    isPrivate: false,
  }, 
  {
    where: {
      isPrivate: true,
      makePublicOn: { [Op.lte]: today },
    }
  }
).then(([affectedCount]) => {
  logger.info(`Number of private updates made public: ${affectedCount}`)
});
```

See https://docs.sequelizejs.com/manual/instances.html#working-in-bulk--creating--updating-and-destroying-multiple-rows-at-once-

Reply:
Thanks for this, I missed the bulk operations part of the docs, should've looked that up earlier, I'll make this change asap.